code/game/objects/effects/roaches.dm

```diff
@@ -1,13 +1,13 @@
 //generic procs copied from obj/effect/alien
-/obj/effect/roach
+/obj/item/weapon/roach
```

Comment:
Why is it a weapon instead of just being an item?

Reply:
it is one now

~~the true answer is that i kinda forgot if the thing would work without being a weapon~~